src/integrator_interface.jl

```diff
@@ -445,6 +445,10 @@ Base.@propagate_inbounds function Base.getindex(A::DEIntegrator, sym)
                 return A.t
             elseif has_sys(A.f) && is_param_sym(A.f.sys, sym)
                 return A.p[param_sym_to_index(A.f.sys, sym)]
+            elseif has_sys(A.f) &&
```

Comment:
Should we just ask the user to write `sys = complete(sys)`?

Reply:
Is this always the case? I am only familiar with Catalyst (and not that familiar with `complete`, but would one always want this to be the case?

Reply:
If you have `sys = complete(sys)`, `sys.x` will no longer namespace.

Reply:
I don't know, I think that if this is correct, then it's a nice thing to add. Even without requiring a complete system.